erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -51,6 +51,15 @@ const BaseStack = (specInput) => {
   };
 
   that.peerConnection = new RTCPeerConnection(that.pcConfig, that.con);
+  let negotiationneededCount = 0;
+  that.peerConnection.onnegotiationneeded = () => {  // one per media which is added
+    let medias = that.audio ? 1 : 0;
+    medias += that.video ? 1 : 0;
+    if (negotiationneededCount % medias === 0) {
+      that.createOffer(true, true);
```

Comment:
This means that now we only create offers por publishing as a response of a negotiationneeded event right?

Reply:
yes, or if we're subscribing and there's no local description set yet.